ScpEepromController.cpp

```diff
@@ -123,17 +122,17 @@ String ScpEepromController::getPassword()
    String password = "";
    for (int i = PW_ADDRESS; i <= PW_LENGTH; i++)
    {
-      scpDebug.println(scpDebug.eeprom, "      ScpEepromController.getPassword: EEPROM Addr: " + String(i));
+      scpDebug.println(scpDebug.eeprom, "ScpEepromController.getPassword: EEPROM Addr: " + String(i));
       password = password + char(EEPROM.read(i));
-      scpDebug.println(scpDebug.eeprom, "      ScpEepromController.getPassword: char " + String(char(EEPROM.read(i))));
+      scpDebug.println(scpDebug.eeprom, "ScpEepromController.getPassword: char " + String(char(EEPROM.read(i))));
    }
    return password;
 }
 
 /**
    * @brief 
    * 
-   * @return password 
+   * @param password 
    */
 void ScpEepromController::setPassword(String password)
```

Comment:
currentPassowortNumber is given as return value. This should be reflected in the return type.
```suggestion
uint32_t ScpEepromController::setPassword(String password)
```

Reply:
I will note this as a future improvement.